src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/EventHandlers/ChangeOfPartyRequestCohortCreatedEventHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿using MediatR;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetCohortSummary;
+using SFA.DAS.CommitmentsV2.Messages.Commands;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.EventHandlers
+{
+    public class ChangeOfPartyRequestCohortCreatedEventHandler : IHandleMessages<ChangeOfPartyRequestCohortCreatedEvent>
+    {
+        public const string ApproveNewEmployerDetailsLevy = "ApproveNewEmployerDetails_Levy";
```

Comment:
Might be worth having the word "Template" appear in the const names, for clarity.